homeassistant/components/flume/sensor.py

```diff
@@ -152,7 +156,11 @@ def update(self):
             self._available = False
             return
         _LOGGER.debug("Successful update of flume sensor: %s", self._name)
-        self._state = self._flume_device.value
+        self._state = self._flume_device.values["current_interval"]
+        self._attributes = {
```

Comment:
Almost there. 👍  Rounding is working but I'm still seeing `current_interval` in the attributes

<img width="441" alt="Screen Shot 2020-07-24 at 9 08 06 PM" src="https://user-images.githubusercontent.com/663432/88451366-da35e900-cdf1-11ea-9ef7-f6f90076c6a2.png">

Reply:
Thanks for the feedback. I believe I caught the missing brackets!